RecoLocalTracker/SiPixelClusterizer/test/gpuClustering.cu

```diff
@@ -174,88 +158,73 @@ int main(void)
   size_t size32 = n * sizeof(unsigned int);
   size_t size16 = n * sizeof(unsigned short);
   size_t size8 = n * sizeof(uint8_t);
-	 
+
   uint32_t nModules=0;
   cuda::memory::copy(d_moduleStart.get(),&nModules,sizeof(uint32_t));
-  
+
   cuda::memory::copy(d_id.get(), h_id.get(), size16);
   cuda::memory::copy(d_x.get(), h_x.get(), size16);
   cuda::memory::copy(d_y.get(), h_y.get(), size16);
   cuda::memory::copy(d_adc.get(), h_adc.get(), size8);
-  cuda::memory::device::zero(d_debug.get(),size32);  
+  cuda::memory::device::zero(d_debug.get(),size32);
   // Launch CUDA Kernels
-
-  
   int threadsPerBlock = 256;
   int blocksPerGrid = (numElements + threadsPerBlock - 1) / threadsPerBlock;
   std::cout
     << "CUDA countModules kernel launch with " << blocksPerGrid
     << " blocks of " << threadsPerBlock << " threads\n";
-		     
+
   cuda::launch(
-	       countModules, 
-	       { blocksPerGrid, threadsPerBlock },
-	       d_id.get(), d_moduleStart.get() ,d_clus.get(),n
-	       );
-  
+               countModules,
+               { blocksPerGrid, threadsPerBlock },
+               d_id.get(), d_moduleStart.get() ,d_clus.get(),n
+               );
+
   threadsPerBlock = 256;
   blocksPerGrid = MaxNumModules;    //nModules;
 
-
-
   std::cout
     << "CUDA findModules kernel launch with " << blocksPerGrid
     << " blocks of " << threadsPerBlock << " threads\n";
 
   cuda::memory::device::zero(d_clusInModule.get(),MaxNumModules*sizeof(uint32_t));
 
   cuda::launch(
-	       findClus,
-	       { blocksPerGrid, threadsPerBlock },
-	       d_id.get(), d_x.get(), d_y.get(),  d_adc.get(),
-	       d_moduleStart.get(),
-	       d_clusInModule.get(), d_moduleId.get(),
-	       d_clus.get(),
-	       d_debug.get(),
-	       n
-	       );
-
+               findClus,
+               { blocksPerGrid, threadsPerBlock },
+               d_id.get(), d_x.get(), d_y.get(),  d_adc.get(),
+               d_moduleStart.get(),
+               d_clusInModule.get(), d_moduleId.get(),
+               d_clus.get(),
+               d_debug.get(),
+               n
+               );
 
   cuda::memory::copy(&nModules,d_moduleStart.get(),sizeof(uint32_t));
   std::cout << "found " << nModules << " Modules active" << std::endl;
```

Comment:
remove?

Reply:
I don't mind keeping the `cout` since this is a just a test file, not one used in production.

Reply:
it's a test!